karapace/schema_backup.py

```diff
@@ -240,9 +240,9 @@ def export(self, export_func) -> None:
             LOG.info("Schema export written to stdout")
         self.close()
 
-    def encode_key(self, key: Optional[Union[JsonData, str]]) -> bytes:
-        if not key:
-            return b""
+    def encode_key(self, key: Optional[Union[JsonData, str]]) -> Optional[bytes]:
+        if key == "null":
```

Comment:
I would prefer null byte for null data. And would solve the literal string "null" in the `_restore_backup_version_1` as that is the source for those.
Essentially means that both internal restore functions would need to handle the key and value encoding, and pass key and value as separate arguments to `_handle_restore_message`.

Reply:
My proposal to use null byte is not feasible. It would make impossible to distinguish between actual null byte payload and tombstone marker. The `null` string is good here as it can be safely recognized as actual null payload.